docs/en/serverless/index.asciidoc

```diff
@@ -1,6 +1,4 @@
-:doctype: book
-
-include::{asciidoc-dir}/../../shared/versions/stack/master.asciidoc[]
+include::{asciidoc-dir}/../../shared/versions/stack/current.asciidoc[]
```

Comment:
I initially used `master` because I thought we wanted to _link_ to the `master` version of stateful docs, but that messed up code blocks using `version` ([example](https://www.elastic.co/guide/en/serverless/current/observability-stream-log-files.html#observability-stream-log-files-step-1-download-and-extract-the-agent-installation-package)).  😬